tests/unittests/conftest.py

```diff
@@ -0,0 +1,360 @@
+                    },
+                    {
+                        "step": "*ignore*",
+                        "failure_type": "test_failure",
+                        "classification": "NONE",
+                        "jira_project": "NONE",
+                        "ignore": "true",
+                    },
+                    {
+                        "step": "affects-version",
+                        "failure_type": "all",
+                        "classification": "Affects Version",
+                        "jira_project": "TEST",
+                        "jira_epic": "!default",
+                        "jira_affects_version": "4.14",
+                        "jira_assignee": "!default",
+                    },
+                    {
+                        "step": "affects-version",
+                        "failure_type": "all",
+                        "classification": "Affects Version",
+                        "jira_project": "TEST",
+                        "jira_epic": "!default",
+                        "jira_affects_version": "4.14",
+                        "jira_assignee": "!default",
+                    },
+                ],
+            },
+        ),
+    )
+    assert os.getenv(FIREWATCH_CONFIG_ENV_VAR)
+
+
+@pytest.fixture
+def assert_artifact_dir_exists(artifact_dir):
+    artifact_dir.mkdir(exist_ok=True, parents=True)
+    assert artifact_dir.is_dir()
+
+
+@pytest.fixture
+def jira_config_path(tmp_path):
+    yield tmp_path.joinpath("jira_config.json")
+
+
+@pytest.fixture(params=JOB_STEP_DIRS_TO_TEST)
+def job_log_step_dirs(request, job_log_dir):
+    yield (job_log_dir / p for p in request.param)
+
+
+@pytest.fixture(params=JOB_STEP_DIRS_TO_TEST)
+def job_junit_step_dirs(request, job_junit_dir):
+    yield (job_junit_dir / p for p in request.param)
+
+
+@pytest.fixture
+def job_log_dir(job_dir):
+    yield job_dir / "logs"
+
+
+@pytest.fixture
+def job_junit_dir(job_dir):
+    yield job_dir / "artifacts"
+
+
+@pytest.fixture(params=JOB_DIR_LOG_ARTIFACT_FILE_NAMES_TO_TEST)
+def job_dir_log_artifact_paths(request, job_log_step_dirs):
+    yield (p / request.param[0] for p in job_log_step_dirs)
+
+
+@pytest.fixture(params=JOB_DIR_JUNIT_ARTIFACT_FILE_NAMES_TO_TEST)
+def job_dir_junit_artifact_paths(request, job_junit_step_dirs):
+    yield (p / request.param[0] for p in job_junit_step_dirs)
+
+
+@pytest.fixture
+def assert_artifact_dir_in_env(monkeypatch, artifact_dir):
+    monkeypatch.setenv(ARTIFACT_DIR_ENV_VAR, artifact_dir.as_posix())
+    assert os.getenv(ARTIFACT_DIR_ENV_VAR)
+
+
+@pytest.fixture
+def artifact_dir(tmp_path):
+    yield tmp_path / "artifacts"
+
+
+@pytest.fixture
+def job_dir(tmp_path, build_id):
+    yield tmp_path / build_id
+
+
+@pytest.fixture(params=BUILD_IDS_TO_TEST, ids=BUILD_IDS_TO_TEST)
+def build_id(request):
+    yield request.param
+
+
+@pytest.fixture(params=DEFAULT_JIRA_PROJECTS_TO_TEST, ids=DEFAULT_JIRA_PROJECTS_TO_TEST)
```

Comment:
`ids` if parametrized is for giving a better name to the parameter test, you use the same names for the parameters and the `ids`, I think you will get the same without setting `ids`